src/effects/VST/VSTEffect.cpp

```diff
@@ -2347,7 +2351,10 @@ intptr_t VSTEffectWrapper::callDispatcher(int opcode,
                                    int index, intptr_t value, void *ptr, float opt)
 {
    // Needed since we might be in the dispatcher when the timer pops
-   wxCRIT_SECT_LOCKER(locker, mDispatcherLock);
+   //wxCRIT_SECT_LOCKER(locker, mDispatcherLock);
+
+   std::lock_guard<std::recursive_mutex> lk(recm);
```

Comment:
I think you can simplify in C++17 with CTAD -- you can leave out the angle brackets